ci/jenkins/jobs/projects-lab.yaml

```diff
@@ -769,6 +839,49 @@
                       default-excludes: true
                       fingerprint: false
                       only-if-success: false
+      - '{name}-{test_name}-for-period':
+          test_name: windows-e2e-userspace
+          node: 'antrea-windows-testbed'
+          description: 'This is for running e2e tests with OVS userspace daemons running as Windows services.'
+          builders:
+            - builder-e2e-win:
+                e2e_type: 'windows-e2e-userspace'
+          branches:
+          - '*/main'
+          included_regions: []
+          cron: 'H H * * *'
+          ignore_post_commit_hooks: false
+          publishers:
+          - email:
+              recipients: projectantrea-dev@googlegroups.com
+          - archive:
+              allow-empty: true
+              artifacts: 'ci/jenkins/*sonobuoy*.tar.gz, whole-e2e-coverage.tar.gz'
```

Comment:
We don't have sonobuoy in windows test, please refer to other windows job configuration.